### ISSUE TITLE
VS Code warning about deprecated command usage

### ISSUE THREAD

**Issue by bjohnso5** [Microsoft/vscode-postgresql#19]
- PGSQL Extension Version: 0.1.0
- VSCode Version: 1.32.3
- OS Version: Windows 10 x64 Version 1803

Steps to Reproduce:
1. Query a table via a configured connection
2. A warning notification is raised in VS Code with the following text:
```
Extension 'ms-ossdata.vscode-postgresql' uses the 'vscode.previewHtml' command which is deprecated and will be removed soon. Please file an issue against this extension to update to use VS Code's webview API.
```
The webview API docs can be found [here](https://code.visualstudio.com/api/extension-guides/webview)

**Comment by fmusayev** [comment]
Almost the same problem
VSCode-Insiders, Version 1.33.0-insider

Can't execute query, end up with error:
`Error: command 'vscode.previewHtml' not found`

**Comment by noinkling** [comment]
I have the same issue as @fmusayev on the release version of 1.33.0

**Comment by resistorsoftware** [comment]
same for me now too... oddly, was working fine until a MacOS update.

**Comment by sockfish** [comment]
same on windows 10 vscode 1.33.0:
Error: command 'vscode.previewHtml' not found

**Comment by fmusayev** [comment]
it is already deprecated and removed, we can't even use this extension after an update, can you please fix this issue asap

**Comment by akamath46297** [comment]
This is now fixed with [Release 0.2.0](https://github.com/Microsoft/vscode-postgresql/releases/tag/v0.2.0)
